### ISSUE TITLE
💡 Dark/Light switch: GET variable in links

### ISSUE THREAD

**Issue by 360fun** [hicetnunc2000/hicetnunc#755]
**PROBLEM**
I would like to be able to decide in which mode (dark/light) people see my links on HEN.

**SOLUTION**
I propose to add a GET variable to whatever link:
https://www.hicetnunc.xyz/tz/tz1cYe4ueZsd6pErnpyXHNDX4tHBjtJy89w3?switch=on
or shorter:
https://www.hicetnunc.xyz/objkt/73232?s=1

**BUTTONS**
In order to make this functionality visible to everybody I was thinking about 2 small buttons to copy the link on the clipboard:
![image](https://user-images.githubusercontent.com/844090/119487462-b157ea80-bd59-11eb-9e3a-c61c15efd27b.png)

**Comment by andrevenancio** [comment]
Personally as a user I think the theme choice is up to me to decide. Either by personal preference, or accessibility reasons. Its not for the author of an NFT to decide the theme IMHO.

**Comment by sf-pixel** [comment]
Agreed with @andrevenancio, next think you know you're scrolling through feed and you see something you like. Click on the OBJKT and you're bilnded by the theme switch from dark to light. Not the best thing to do to your users
